patterns/ecs-v1/aws

```diff
@@ -1,14 +1,22 @@
-S3_REQUEST_LINE (?:%{WORD:verb} %{NOTSPACE:request}(?: HTTP/%{NUMBER:httpversion})?|%{DATA:rawrequest})
+# TODO TEMP: revert URIQUERY
+URIQUERY [A-Za-z0-9$.+!*'|(){},~@#%&/=:;_?\-\[\]<>]*
 
-S3_ACCESS_LOG %{WORD:owner} %{NOTSPACE:bucket} \[%{HTTPDATE:timestamp}\] %{IP:clientip} %{NOTSPACE:requester} %{NOTSPACE:request_id} %{NOTSPACE:operation} %{NOTSPACE:key} (?:"%{S3_REQUEST_LINE}"|-) (?:%{INT:response:int}|-) (?:-|%{NOTSPACE:error_code}) (?:%{INT:bytes:int}|-) (?:%{INT:object_size:int}|-) (?:%{INT:request_time_ms:int}|-) (?:%{INT:turnaround_time_ms:int}|-) (?:%{QS:referrer}|-) (?:"?%{QS:agent}"?|-) (?:-|%{NOTSPACE:version_id})
+S3_REQUEST_LINE (?:%{WORD:[http][request][method]} %{NOTSPACE:[url][original]}(?: HTTP/%{NUMBER:[http][version]})?)
```

Comment:
Beats seems to not split this part into fields.

Reply:
Yes, the field `http.version` should only include the actual version number, not the word "http"

Reply:
Thanks, what I meant is that this whole `S3_REQUEST_LINE` does not get parsed out by beats.
Beats captures these (3) fields under `aws.s3access.request_uri` (but e.g. 0there's no `http.version` captured), 
LS does not capture the line under `aws.s3access.request_uri` and instead parses the rest.
We could also capture `aws.s3access.request_uri` but I am not sure it makes sense other than beats compatibility.

Reply:
Ah right. Matching Beats as much as possible is a good goal.

In this case however, my recommendation would instead to be to match the ECS recommendation, as `aws.s3access.request_uri ` is clearly a field that was overlooked in the Beats module.

2 reasons why:

* If I were a user using both the Logstash and Beats implementations, seeing this discrepancy I would finish the normalization of the Beats data in a custom pipeline, rather than de-normalizing the Logstash one, to put that URI back in a custom field and match Beats.
* If you match Beats on this, now we have 2 data sources that aren't quite normalized to match ECS, and therefore needs adjustments.